release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1646,6 +1659,8 @@ With this release, the installation program verifies whether a standard port gro
 [id="ocp-4-17-management-console-bug-fixes_{context}"]
 ==== Management Console
 
+* Previously, the *Cluster overview* page included a `View all steps in documentation` link that resulted in a 404 error for ROSA and OSD clusters. With this update, the link does not appear for ROSA and OSD clusters. (link:https://issues.redhat.com/browse/OCPBUGS-37054[*OCPBUGS-37054*])
```

Comment:
```suggestion
* Previously, the *Cluster overview* page included a `View all steps in documentation` link that resulted in a 404 error for {product-rosa} and {product-dedicated} clusters. With this update, the link does not appear for {product-rosa} and {product-dedicated} clusters. (link:https://issues.redhat.com/browse/OCPBUGS-37054[*OCPBUGS-37054*])
```
Better check if this note covers both "{product-title} (ROSA) (classic architecture)" and "ROSA with HCP" versions.